db/migrate/20180915035427_add_accessbility_text_field_to_shelters.rb

```diff
@@ -0,0 +1,5 @@
+class AddAccessbilityTextFieldToShelters < ActiveRecord::Migration[5.1]
+  def change
```

Comment:
Both of your migrations should be in the same file since they are co-related.  Instead of using a `change` method, you can put both the add_column and data modification in a `up` method, and add a `down` method to remove_column.

See for more info: https://railsguides.net/change-data-in-migrations-like-a-boss/

Reply:
I did it this way first, then re-thought it and separated them on the idea that they're two separate operations, as well as looking back in the migration history and seeing the separate `20170909082743_backfill_special_needs.rb` migration.

I'll rework it the way I was doing it before.